docs/Salesforce-batchsink.md

```diff
@@ -30,6 +30,8 @@ A Salesforce connected application must be created in order to get a client secr
 
 **Connect Timeout:** Maximum time in milliseconds to wait for connection initialization before time out.
 
+**Proxy URL:** Proxy URL. Must contain a protocol, address and port.
```

Comment:
Can the other changes be reverted in this md file? Looks like formatting changes?

Reply:
other changes are related to removing <br> tag and use double space instead. We can keep those changes.